README.md

```diff
@@ -124,13 +125,21 @@ Then power on the device.
 4. Connect the power adapter to the Power Jack [J16].
 5. The device will automatically power on in Force Recovery Mode.
 
-**Jetson AGX Orin:**
+**Jetson AGX Orin 32GB Devkit:**
 
 - Make sure you put the Type-C plug of the cable into the USB Type-C port next to 40-pin connector for flashing.
 - While holding the middle Force Recovery button, insert the USB Type-C power supply plug into the USB Type-C port above the DC jack.
 - This will turn on the Jetson dev kit in Force Recovery Mode.
 - HOLD DOWN UNTIL you hear the fan and get a usb connection popup on your connected PC
 
+**Jetson AGX Orin 64GB Devkit:**
+
+- Make sure you put the Type-C plug of the cable into the USB Type-C port next to 40-pin connector for flashing.
```

Comment:
- specify USB C data cable vs. power cable

Reply:
thanks @alanb128 , clarified for both in https://github.com/balena-os/jetson-flash/pull/165/commits/d471fc65f4b6770bde5f2eae5202a86b5f91bf6a